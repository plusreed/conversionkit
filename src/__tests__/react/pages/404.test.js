import React from 'react';
import renderer from 'react-test-renderer'
import NotFoundPage from '../../../app/pages/404';

it('matches snapshot', () => {
  const tree = renderer
    .create(<NotFoundPage />)
    .toJSON()

  expect(tree).toMatchSnapshot()
});
