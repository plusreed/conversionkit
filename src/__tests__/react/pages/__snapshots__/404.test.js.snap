// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div
  className="p-4"
>
  <h1
    className="font-bold text-2xl"
  >
    404
  </h1>
  <p
    className="font-normal text-md"
  >
    The page you requested could not be found.
  </p>
</div>
`;
